cpu/nrf5x_common/periph/timer.c

```diff
@@ -106,6 +107,30 @@ int timer_set_absolute(tim_t tim, int chan, unsigned int value)
     return 0;
 }
 
+int timer_set_periodic(tim_t tim, int chan, unsigned int value, uint8_t flags)
+{
+    /* see if channel is valid */
+    if (chan >= timer_config[tim].channels) {
+        return -1;
+    }
+
+    ctx[tim].flags |= (1 << chan);
+    ctx[tim].is_periodic |= (1 << chan);
+    dev(tim)->CC[chan] = value;
+    (void)flags;
```

Comment:
```suggestion
```

it's uses right on the next line

Reply:
Thanks, that was a leftover from experimentation.